pep-0708.rst

```diff
@@ -0,0 +1,849 @@
+  system it's hard to even know where to start to list them.
+
+
+Only recommend installers offer explicit configuration
+------------------------------------------------------
+
+One idea that has come up is to essentially just implement the explicit
+configuration and don't make any other changes to anything else. The specific
+proposal for a mapping policy is what actually inspired explicit configuration
+option, and created a file that looked something like:
+
+  .. code-block:: JSON
+
+    {
+      "repositories": {
+        "PyTorch": ["https://download.pytorch.org/whl/nightly"],
+        "PyPI": ["https://pypi.org/simple"]
+      },
+      "mapping": [
+        {
+          "paths": ["torch*"],
+          "repositories": ["PyTorch"],
+          "terminating": true
+        },
+        {
+          "paths": ["*"],
+          "repositories": ["PyPI"]
+        }
+      ]
+    }
+
+The recommendation to have explicit configuration pushes the decision on how to
+implement that onto each installer, allowing them to choose what works best for
+their users.
+
+Ultimately only implementing some kind of explicit configuration was rejected
+because by its nature it's opt in, so it doesn't protect average users who are
+least capable to solve the problem with the existing tools, by adding additional
+protections alongside the explicit configuration, we are able to protect all
+users by default.
+
+Additionally, relying on only explicit configuration also means that every end
+user has to resolve the same problem over and over again, even in cases like
+mirrors of PyPI, Piwheels, PyTorch, etc. In each and every case they have to sit
+there and make decisions (or find some example to cargo cult) in order to be
+secure. Adding extra features into the mix allows us to centralize those
+protections where we can, while still giving advanced end users the ability to
+completely control their own destiny.
+
+
+Scopes à la npm
+---------------
+
+There's been some suggestion that scopes similar to how npm has implemented them
+may ultimately solve this. Ultimately scopes do not change anything about this
+problem. As far as I know scopes in npm are not globally unique, they're tied to
+a specific registry just like unscoped names are. However what scopes do enable
+is an obvious mechanism for grouping related projects and the ability for a user
+or organization on npm.org to claim an entire scope, which makes explicit
+configuration significantly easier to handle because you can be assured that
+there's a whole little slice of the namespace that wholly belongs to you, and
+you can easily write a rule that assigns an entire scope to a specific non
+public registry.
+
+Unfortunately, it basically ends up being an easier version of the idea to only
+use explicit configuration, which works ok in npm because its not particularly
+common for people to use their own registries, but in Python we encourage you to
+do just that.
+
+
+Open questions
```

Comment:
```suggestion
Open Questions
```

Title case for top-level headings is the convention both in this PEP and with PEPs generally

Reply:
My editor, for whatever reason, keeps trying to correct caps like this into lowercase versions. I assume just some impedance mismatch between using a code editor to write prose. Thanks for catching one I missed to correct manually.